ssg/entities/common.py

```diff
@@ -128,7 +133,8 @@ def derive_id_from_file_name(filename):
 
 def dump_yaml_preferably_in_original_order(dictionary, file_object):
     try:
-        return yaml.dump(dictionary, file_object, indent=4, sort_keys=False)
+        return yaml.dump(dictionary, file_object, indent=4, sort_keys=False,
```

Comment:
This is a nice improvement. I got a speed up about 6 seconds when building RHEL 9 content.

Can you investigate if this change can be consolidated with `ssg/yaml.py`?

Reply:
I updated to import `yaml_Dumper` from `ssg/yaml.py`.